### PR TITLE
Nc news ticket 8

### PR DESCRIPTION
**Pull request by johnnywalker-git**
Added functionality for the user to update votes on an article by a given amount.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -123,4 +123,27 @@ describe("Ticket 5.", () => {
...
+    .patch("/api/articles/43492")
+    .send({ "inc_votes" : 7 })
+    .expect(404)
+  })
+  test("Article should respond with the correct error when passed an invalid article ID TYPE", () => {
+    return request(app)
+    .patch("/api/articles/abcde")
+    .send({ "inc_votes" : 14 })
+    .expect(400)
+  })
```
Comment (RichardNeat) [inline __tests__/app.test.js:191]: Could we also consider an error if the input body is malformed, either invalid value or missing keys for example.

File: controllers/articles.controller.js [hunk 2]
```diff
@@ -13,6 +13,18 @@ exports.getAllArticles = (req, res, next) => {
    })
 }
 
+exports.updateArticleVotes = (req, res, next) => {
+   const articleNo = req.params
+   const VoteAmount = req.body
+   updateVotes(articleNo, VoteAmount)
+   .then((article) => {
+      res.status(200).send(article)
```
Comment (RichardNeat) [inline controllers/articles.controller.js]: Lets make sure to send the result back on an object - you will likely need to update your tests to do this so make sure it fails first 😉

**Review by RichardNeat — APPROVED** [review]
Nice one! Just a couple of suggested changes below before merging please. No need to resubmit :)

File: controllers/articles.controller.js (hunk 2 above)
Comment (johnnywalker-git) [reply controllers/articles.controller.js]: That was speedy! Thanks Rich, i'll take a look!

**Pull request merged**